dls-controls/malcolmjs
# Feature/nav bar tweak #335

## Description

Changed navbar so dropdown appears to the right of the element and three dots appear to select a new child (if the last element has any). Required some tweaks to the navigation reducer so each nav element now tracks its parent.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/gui
- use the navbar to navigate around

## Agile board tracking

connect to #335 
closes #335
